.github/workflows/package_all.yaml

```diff
@@ -20,7 +27,7 @@ jobs:
           fetch-depth: 0 # Required because of release/get_version.py
       - name: "Build package"
         run: |
-          ./release/package/run.sh package centos-7
+          ./release/package/run.sh package centos-7 ${{ github.event.inputs.build_type }}
```

Comment:
When do we actually pass the `github.event.inputs.build_type`?

Reply:
github does based on the spec above